internal/signer/linux/util/cert_util.go

```diff
@@ -0,0 +1,96 @@
+// Cert_util provides helpers for working with certificates via PKCS11
+package util
+
+import (
+	"crypto"
+	"errors"
+	"io"
+
+	"github.com/google/go-pkcs11/pkcs11"
+)
+
+// Cred returns a Key wrapping the first valid certificate in the pkcs11 module
+// matching a given slot and label.
+func Cred(pkcs11_module string, slot_uint32_str string, label string) (*Key, error) {
+	module, err := pkcs11.Open(pkcs11_module)
+	if err != nil {
+		return nil, err
+	}
+	slot_uint32, err := ParseHexString(slot_uint32_str)
+	if err != nil {
+		return nil, err
+	}
+	kslot, err := module.Slot(slot_uint32, pkcs11.Options{})
```

Comment:
I think we should consider passing a PIN to the underlying PKCS#11 device, otherwise the device may be unusable

Reply:
Should be easy enough to expand the `Cred` parameters, but maybe we should wrap the `pkcs11.Options` in a new type?

Reply:
Thanks for the review Carl! Those are great suggestions. I will create a Github issue to track your suggestion after this initial PR is in. We can revisit this once we begin the work to support external Linux and other pkcs11 modules.